.sanity-ansible-ignore-2.9.txt

```diff
@@ -0,0 +1,47 @@
+plugins/module_utils/network_lsr/__init__.py shebang!skip
+plugins/module_utils/network_lsr/argument_validator.py shebang!skip
+plugins/module_utils/network_lsr/myerror.py shebang!skip
```

Comment:
The shebangs can be removed from these files and then they don't need to be skipped.